prisma/schema.prisma

```diff
@@ -246,8 +246,10 @@ model NewApplication {
   disabilityCertificationDate          DateTime                             @map("disability_certification_date") @db.Date
   patientCondition                     PatientCondition                     @map("patient_condition")
   mobilityAids                         MobilityAid[]                        @map("mobility_aids")
+  otherMobilityAids                    String?                              @map("other_mobility_aids") @db.VarChar(255)
   otherPatientCondition                String?                              @map("other_patient_condition") @db.VarChar(255)
   temporaryPermitExpiry                DateTime?                            @map("temporary_permit_expiry") @db.Date
+  permitType                           PermitType                           @default(PERMANENT) @map("permit_type")
```

Comment:
See above https://github.com/uwblueprint/richmond-centre-for-disability/pull/192/files#r869824458